poverty_info.rb

```diff
@@ -0,0 +1,47 @@
+require 'pry'
+
+STATE_FILES = { "Washington" => "povWA.txt", "South Dakota" => "povSD.txt"}
+
+def print_child_poverty_info(state)
```

Comment:
refactored my Ruby code to have three methods- getting the min, getting the max, and printing it out.